lib/note-toolbar/index.tsx

```diff
@@ -171,4 +172,8 @@ export class NoteToolbar extends Component {
   };
 }
 
-export default NoteToolbar;
+const mapStateToProps = ({ ui: { note } }) => ({
+  note,
+});
```

Comment:
it's so good to see changes like this where we're moving from passing props _through_ components to getting the props from the global Redux store 👍

Reply:
For sure.  As we can start deconstructing components and moving logic out to utils the codebase will be more manageable.